docs/content/doc/advanced/config-cheat-sheet.en-us.md

```diff
@@ -77,6 +77,8 @@ Values containing `#` or `;` must be quoted using `` ` `` or `"""`.
 - `FEED_MAX_COMMIT_NUM`: **5**: Number of maximum commits shown in one activity feed.
 - `GRAPH_MAX_COMMIT_NUM`: **100**: Number of maximum commits shown in the commit graph.
 - `DEFAULT_THEME`: **gitea**: \[gitea, arc-green\]: Set the default theme for the Gitea install.
+- `THEMES`:  **gitea,arc-green**: Allow users select personalized themes
+  regardless of the value of `DEFAULT_THEME`.
```

Comment:
This description is different [here](https://github.com/go-gitea/gitea/blob/82e08a3364195b515a005180c2bdc08e78aac208/custom/conf/app.ini.sample#L91), I think they should be made the same.

Reply:
Oops... this was merged already